maven-release-manager/src/main/java/org/apache/maven/shared/release/scm/DefaultScmRepositoryConfigurator.java

```diff
@@ -164,7 +164,9 @@ public ScmRepository getConfiguredRepository(String url, ReleaseDescriptor relea
                 svnRepo.setBranchBase(branchBase);
             }
         }
-
+        if (!releaseDescriptor.isInteractive()) {
```

Comment:
Do we need such condition?
Maybe simply set  - what is default value in scm provider?

Reply:
Default value is true, the condition is necessary to make the tests pass, as they rely on the fact that there is no interaction with the mocked provider in some cases.